app/controllers/images_controller.rb

```diff
@@ -1,13 +1,15 @@
+# frozen_string_literal: true
+
 class ImagesController < ActionController::Base
   PUBLIC_MOUNTS = {
-    'User' => :image,
-  }
+    'User' => :image
+  }.freeze
 
   def show
     http_cache_forever(public: true) do
       model_name = params[:model_name]
       attribute_name = PUBLIC_MOUNTS[model_name]
-      head :not_authorized and return unless attribute_name
+      head(:not_authorized) && return unless attribute_name
```

Comment:
nope, using `and` in this case is a rails best practice

Reply:
Personally I avoid `and` and `or` as the subtle differences to `&&` and `||` can easily lead to unintended behavior. In particular in a code base edited by many people of which some may not be so familiar with the peculiarities of Ruby.

Reply:
well, in this case `and` is the correct ruby keyword while `&&` is not

the proper version without `and` is

```
unless attribute_name
  head(:not_authorized)
  return
end
```